src/Containers/MinimalContainers/tests/test_ConstantSizeMatrix.cpp

```diff
@@ -28,8 +28,16 @@ TEST_CASE("ConstantSizeMatrix basics", "[containers]")
   CHECK(cmat.capacity() == 32);
 
   CHECK_THROWS(cmat.resize(1, 33));
+  CHECK_THROWS(cmat.resize(34, 2));
 
-  CHECK_THROWS(cmat.resize(2, 9));
+  CHECK_NOTHROW(cmat.resize(2, 9));
+  CHECK(cmat.capacity() == 32);
+
+  // If the new dimensions are not commensurable with the old ones,
+  // the capacity could decrease, even though the size of the underlying
```

Comment:
This comment is confusing since directly below you show the capacity doesn't decrease. I think what you mean is the maximum size consistent with the resize arguments can decrease.

Reply:
The comment is no longer valid - the capacity does not decrease.  I'll remove it.